.github/workflows/conformance-clustermesh.yaml

```diff
@@ -0,0 +1,504 @@
+name: ClusterMesh
```

Comment:
Let's add the trigger phrase to the test name so it's easier to find from a PR's actions status overview.

```suggestion
name: ClusterMesh (ci-clustermesh)
```

Reply:
We should add some linter for this. I always forget.